<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>633</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>828</width>
    <height>633</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>828</width>
    <height>633</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>9999</width>
     <height>9999</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar {background: none; color:white;}
QTabBar::tab {height: 55px; width: 200px; color: f0f0ff;}
QTabBar::tab::selected {background: #a0a0a0;}
QTabWidget::pane {border: 0;}
</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_5">
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #444444;</string>
    </property>
    <attribute name="title">
     <string>Config 수정</string>
    </attribute>
    <widget class="QLabel" name="spot_name_lbl">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>90</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>사업장 이름</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="spot_id">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff; color: black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="spot_name">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>90</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff; color: black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="spot_id_lbl">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>40</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>사업장ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black; background: #999;</string>
     </property>
     <property name="text">
      <string>적용</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_pwd">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff; color: black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_pwd_lbl">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>90</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>계정 PWD</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_id_lbl">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>40</y>
       <width>114</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>계정 ID </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_id">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff; color: black;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <property name="styleSheet">
     <string notr="true">background: #444444;</string>
    </property>
    <attribute name="title">
     <string>Setting 수정</string>
    </attribute>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black; background: #999;</string>
     </property>
     <property name="text">
      <string>적용</string>
     </property>
    </widget>
    <widget class="QLabel" name="api_path_lbl">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>40</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>api 주소</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="save_path">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>40</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff; color: black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="api_path">
     <property name="geometry">
      <rect>
       <x>547</x>
       <y>40</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff; color: black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="save_path_lbl">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>40</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>저장 경로</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="logo_btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>9</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
border: none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
